contracts/multisig-prover/src/test/mocks/monitoring.rs

```diff
@@ -0,0 +1,50 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Binary, Deps, DepsMut, Env, MessageInfo, Response, StdError, StdResult};
+use cw_storage_plus::Map;
+
+use monitoring::msg::QueryMsg;
+use multisig::worker_set::WorkerSet;
+
+pub fn instantiate(
+    _deps: DepsMut,
+    _env: Env,
+    _info: MessageInfo,
+    _msg: monitoring::msg::InstantiateMsg,
+) -> Result<Response, StdError> {
+    Ok(Response::default())
+}
+
+#[cw_serde]
+pub enum ExecuteMsg {
+    RegisterActiveWorkerSet { next_worker_set: WorkerSet },
+}
+
+pub fn execute(
+    deps: DepsMut,
+    _env: Env,
+    info: MessageInfo,
+    msg: ExecuteMsg,
+) -> Result<Response, StdError> {
+    match msg {
+        ExecuteMsg::RegisterActiveWorkerSet { next_worker_set } => {
+            register_active_worker_set(deps, info.sender, next_worker_set);
```

Comment:
I don't think you actually need to do any of this. I think you can just return `Ok(Response::new())`

Reply:
I am doing the same thing inside the function:
```
pub fn set_active_worker_set(
    deps: DepsMut,
    info: MessageInfo,
    next_worker_set: WorkerSet,
) -> Result<Response, ContractError> {
    ACTIVE_WORKERSET_FOR_PROVER.save(deps.storage, info.sender, &(next_worker_set))?;
    Ok(Response::new())
}
```

Reply:
yes but why do you need to actually store anything here? This is the mock and I don't think the query is being called, unless I am mistaken. You just need the execute msg to not error, right?

Reply:
Its done.